gen/src/writer/ucd/segment_tests.rs

```diff
@@ -32,10 +32,20 @@ fn emit_grapheme_cluster_break_test_data(dir: &Path) {
             ..
         } = *case;
 
+
         for (i, ch) in chars.iter().enumerate() {
-            let ref gcb = char_gcbs[i];
+            let gcb = match char_gcbs[i].as_str() {
+                // Corresponds to \p{Extended_Pictographic}, not an actual grapheme cluster property
+                "ExtPict" => continue,
+
+                "Extend_ExtCccZwj" => "Extend",
+                "ZWJ_ExtCccZwj" => "ZWJ",
```

Comment:
👍 for this. I had this issue when I was trying to get my fork working for Unicode 11.0, this is a clever fix!

Out of curiosity, there seems to be no documentation about Extend_ExtCccZwj and ZWJ_ExtCccZwj (it seems they only appear in grapheme cluster break test data). Any idea of the purpose of these two values?

I noticed @behnam was also asking about this in http://www.unicode.org/review/pri372/ .

Reply:
I'm not entirely sure where these names came from, in the case of `ZWJ` I noticed that all of them were renamed to `ZWJ_ExtCccZwj` so it seemed to make sense to just map the value back to `ZWJ`

In the case of `Extend` though there are both `Extend` and `Extend_ExtCccZwj` in the 11.0 test file,  the choice of which are used seems to depend on the codepoint and not the GB# rule. A bit of grepping + sort showed the different rules and character combos

```
[9.0] EMOJI MODIFIER FITZPATRICK TYPE-6 (Extend)
[0.2] COMBINING GRAPHEME JOINER (Extend)
[4.0] COMBINING GRAPHEME JOINER (Extend)
[9.0] COMBINING GRAPHEME JOINER (Extend)

[4.0] COMBINING DIAERESIS (Extend_ExtCccZwj)
[9.0] COMBINING DIAERESIS (Extend_ExtCccZwj)
[0.2] COMBINING GRAVE ACCENT (Extend_ExtCccZwj)
[4.0] COMBINING GRAVE ACCENT (Extend_ExtCccZwj)
[9.0] COMBINING GRAVE ACCENT (Extend_ExtCccZwj)
```

I don't know why they're distinguished myself, they're all still `Extend` in https://www.unicode.org/Public/11.0.0/ucd/auxiliary/GraphemeBreakProperty.txt